### PR TITLE
Webpack packaging

### PR DESCRIPTION
**Pull request by xbill82**
This PR drops Grunt and sets Webpack as default package bundler for browser use.
Builds are launched via

    npm run build

and builds are dropped into `dist/`.

Built files still need some testing IMHO, since some external libraries are excluded (`noParse: [/wsnode\.js/, /crypto/, /buffer/, /node-uuid/]`) and I'm not sure everything works fine.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,22 +28,29 @@
     "bluebird": "3.4.1",
     "bufferutil": "^1.2.1",
     "node-uuid": "1.4.7",
+    "ora": "^0.3.0",
     "utf-8-validate": "^1.2.1",
+    "webpack": "^1.13.1",
```
Comment (dbengsch) [inline package.json]: shouldn't webpack be a devDep ?

**Comment by stafyniaksacha** [comment]
LGTM

File: package.json (hunk 1 above)
Comment (xbill82) [inline package.json]: YEP

**Comment by dbengsch** [comment]
+1

**Comment by jenow** [comment]
+1

**Pull request merged**